template/Dockerfile

```diff
@@ -52,15 +66,17 @@ RUN rm -rf /usr/local/tomcat/webapps/* && \\
   mkdir -p /usr/local/tomcat/temp && \\
   mkdir -p /usr/local/xwiki/data && \\
   curl -fSL "\${XWIKI_URL_PREFIX}/xwiki-platform-distribution-war-\${XWIKI_VERSION}.war" -o xwiki.war && \\
-  echo "\$XWIKI_DOWNLOAD_SHA256 xwiki.war" | sha256sum -c - && \\
+  echo "\$XWIKI_DOWNLOAD_SHA256  xwiki.war" | sha256sum -c - && \\
```

Comment:
why the extra space? typo? :)

Reply:
In alpine, the extra space is required for the sha256 to work. Without space it was failing

Reply:
@ASHISH932 I think we're not talking about the same thinf. This is just the echo command. I don't see why it would require 2 spaces, it doesn't make sense to me :)

Reply:
I know that felt weird to me also, but that's how it works. See this issue.
https://github.com/gliderlabs/docker-alpine/issues/174